tests/hooks/run-negative-corpus.sh

```
@@ -0,0 +1,233 @@
+    if [ ! -f "$f" ]; then
+        printf 'run-negative-corpus: fixture not found: %s\n' "$f" >&2
+        exit 1
+    fi
+    fixtures+=("$f")
+fi
+
+# ----- Per-fixture run ------------------------------------------------------
+
+total_pass=0
+total_fail=0
+total_skip=0
```

Shellcheck (0.11.0)

[warning] total_skip appears unused. Verify use (or export if used externally).

(SC2034)